app/views/application-submitted.html

```diff
@@ -1,3 +1,5 @@
+<meta name="robots" content="noindex, nofollow">
```

Comment:
this should be added to header, using the `{% block head %}{% endblock %}` in the `/app/views/govuk_template.html` ln28